app/javascript/controllers/hotdog_controller.js

```diff
@@ -2,7 +2,34 @@ import { Controller } from "@hotwired/stimulus"
 
 // Connects to data-controller="hotdog"
 export default class extends Controller {
-  static targets = ["rightPane", "leftPane", "hotdogCollapse", "hotdogExpand", "burgerCollapse", "burgerExpand", "burgerCollapsible"];
+  static targets = ["rightPane", "leftPane", "hotdogCollapse", "hotdogExpand", "burgerCollapse", "burgerExpand", "burgerCollapsible", "evaluationForm"];
+
+  connect() {
+    this.moveEvaluationForm();
+
+    this.resizeHandler = this.moveEvaluationForm.bind(this);
+    window.addEventListener('resize', this.resizeHandler);
+  }
+
+  disconnect() {
+    window.removeEventListener('resize', this.resizeHandler);
+  }
+
+  moveEvaluationForm() {
+    const evaluationFormContent = document.getElementById('evaluation-form-container');
+    const mobileContainer = document.getElementById('mobile-evaluation-form');
+    const desktopContainer = document.getElementById('desktop-evaluation-form');
+    
+    const breakpoint = 640; // tablet breakpoint
+    const isMobile = window.innerWidth < breakpoint;
+    const isDesktop = window.innerWidth >= breakpoint;
+    
+    if (isMobile && evaluationFormContent.parentElement !== mobileContainer) {
+      mobileContainer.appendChild(evaluationFormContent);
+    } else if (isDesktop && evaluationFormContent.parentElement !== desktopContainer) {
+      desktopContainer.appendChild(evaluationFormContent);
```

Comment:
it's sorta surprising to me that `appendChild` doesn't re-add duplicate elements each time the function is called, but it seems like it works at least in the browser I'm testing on (Brave)